docker/conf/docker-compose-win.yml.dist

```diff
@@ -5,8 +5,9 @@ services:
         container_name: shopsys-framework-postgres
         volumes:
             - pgdata:/var/lib/postgresql/data
+            - ./project-base/docker/postgres/postgres.conf:/var/lib/postgresql/data/postgresql.conf
         environment:
-            - PGDATA=/var/lib/postgresql/data
```

Comment:
it is strange to me that Windows is the only platform where the `PGDATA` was set actually, how come?

Reply:
Reason for it was "added docker-compose for windows operating systems with defined volumes"
I have no idea either where it come from :-D